.travis.yml

```diff
@@ -11,4 +11,4 @@ script: cd scripts/tests; ./scripts_regression_tests.py --machine centos7-linux
 # it's pushed
 branches:
   only:
-  - master
+  - nuopc-cmeps
```

Comment:
This looks suspicious. We want travis to run on master, right?

Reply:
fixed